.changelog/2663.internal.md

```diff
@@ -0,0 +1 @@
+Add data about known addresses
```

Comment:
Filename is incorrect. Requires tweak after running make changelog.

Reply:
oh shoot was it wrong number?

Reply:
yeah looks like it was created manually with some random number

Reply:
Sorry about that, I have no idea where that number came from.

Reply:
I think we can open a PR to rename it